frontends/web/src/routes/account/receive/receive.tsx

```diff
@@ -167,8 +177,7 @@ class Receive extends Component<Props, State> {
         const forceVerification = secureOutput.hasSecureOutput && !secureOutput.optional;
         const enableCopy = !forceVerification;
 
-        // TODO: unified-accounts, let user cycle through address types
-        const currentAddresses = receiveAddresses[0];
+        const currentAddresses = receiveAddresses[addressType];
```

Comment:
there is 3 sets of receive addresses at the moment

Reply:
depends on the keystore. bb02 does not support legacy, so there it's only two.
software keystore supports all but is not present in the release.
bb01 has no unified accounts, so there it's only one.